.gitignore

```diff
@@ -1,6 +1,10 @@
 # Dependency directory
 node_modules
 
+package-lock.json
+
+build
```

Comment:
We actually commit the build dir on purpose, because the rest of our stack pulls this repo in from npm and gets the abis from them